src/iuse.cpp

```diff
@@ -9693,14 +9693,14 @@ int iuse::gobag_personal( player *p, item *it, bool, const tripoint & )
     return gobag( p, it, true );
 }
 
-int iuse::magnesium_tablet( player *p, item *it, bool, const tripoint & )
+int iuse::melatonin_tablet( player *p, item *it, bool, const tripoint & )
```

Comment:
Why do we need an iuse for this?

Reply:
Presumably for the message if you've already taken it. Not a super strong justification, but that's what I can come up with.

Reply:
I think that's information we shouldn't give to the player. We don't warn for excessive use of other drugs.

Reply:
Alright, will look into cutting that.

Reply:
It's worth noting that additional doses of melatonin just generally don't do anything